webapp/scripts/functions.js

```diff
@@ -82,6 +82,28 @@ module.exports = {
     return grass(mapset, `db.describe -c table=${layer}`).trim().split('\n').filter(col => col.match(/DOUBLE PRECISION|INTEGER/)).filter(col => !col.match(/cat/i))
   },
 
+  /**
+   * Returns univariate statistics on selected table column for a GRASS vector map.
+   * @param {string} mapset 
+   * @param {string} map 
+   * @param {string} column 
+   */
+  getUnivar(mapset, map, column) {
+    return grass(mapset, `v.db.univar -e -g map=${map} column=${column}`).trim().split('\n')
```

Comment:
It would be nice if ithis function returned an object with keys/values like `{ min: 0, max: 10, ... }` instead of an array of strings that still need to be parsed.

Reply:
In `module_2.js, line 113-126` the PDF output needs to use this array of strings.

Reply:
And wouldn't `stats.min` be more readable than `stats[1]`?

Reply:
Also, using an object is the more robust approach because you don't have to rely on a given sequence of array items

Reply:
Agree

Reply:
Do you want to fix it for this PR or shall we deal with it separately?